handlers.go

```diff
@@ -245,10 +228,10 @@ func mkSQLRow(N int) []interface{} {
 // the return value is a list of the retrieved records.
 func runQuery(db dbType,
 		qstring string,
-		ivals []interface{}) ([]*map[string]interface{}, error) {
+		ivals []interface{}) ([]map[string]interface{}, error) {
 	log.Debugf("query = %s", qstring)
 
-	ret := make([]*map[string]interface{}, 0, maxRecs)
+	ret := make([]map[string]interface{}, 0, maxRecs)
```

Comment:
I would not recommend provide a maxRecs=1000 as capacity here. Go slice is self-extendable, it will re-allocate the the underlaying array if it reaches its maximum capacity. If you provide maxRecs=1000 for `make`, the underlaying array will have a size of 1000 at init, occupying a lot of memory.
Maybe you can just `ret := make([]map[string]interface{}, 0)` or decrease maxRecs.